app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/cloudBackendApi.ts

```diff
@@ -1,11 +1,12 @@
 /** @file Module containing the API client for the Cloud backend API.
  *
- * Each exported function in the {@link Backend} in this module corresponds to an API endpoint. The
+ * Each exported function in the {@link CloudBackendAPI} in this module corresponds to an API endpoint. The
```

Comment:
Paweł's comment here - if we are changing one of the files to `localBackend`, this one can be changed to `remoteBackend` to keep it even more synced